src/Processors/QueryPlan/Optimizations/reuseStorageOrderingForWindowFunctions.cpp

```diff
@@ -102,10 +102,7 @@ size_t tryReuseStorageOrderingForWindowFunctions(QueryPlan::Node * parent_node,
             limit);
 
     if (order_info)
-    {
         read_from_merge_tree->setQueryInfoInputOrderInfo(order_info);
-        sorting->convertToFinishSorting(order_info->order_key_prefix_descr);
```

Comment:
It shouldn't be necessary any more, - we should get sort description from input stream on sorting step. @novikd please could you have a look?